### DISCUSSION TITLE
Firmware Update OTA bricht immer wieder ab.

### DISCUSSION THREAD

**Discussion by FischerHD** [tbnobody/OpenDTU#1750] [Show and tell]
Möchte gerade von der Firmware v23.10.9 auf die neue v24.2.12 Upgraden aber der Upload Prozess vom 0-100% geht sehr schnell und dann bekomme ich immer den Fehler:   [HTTP ERROR] Bad Request
Was kann das Problem sein oder gibt es Lösungsvorschläge ?

**Comment by Omega13x** [comment]
Die Firmware hatte ein Problem. Musst du seriel flashen.

**Reply by FischerHD** [reply]
Also die Firmware v23.10.9 hatte das Problem, wenn ich das richtig verstanden habe. Denn ich habe jetzt etliche Firmware Versionen von v23.12.31 bis zur aktuellen ausprobiert und immer den gleichen Fehler. Danke Nochmals für die Info.

**Reply by tbnobody** [reply]
Nein.... Im März 2023 wurde das Partitionsschema geändert. Siehe: https://www.opendtu.solar/firmware/howto/upgrade_partition/

**Comment by 3DJupp** [comment]
Dürfte nicht immer ein Problem der Firmware sein. Oft ist es bei mehreren Accesspoints und Problemen mit dem Roaming auch ein Thema.
Ich habe aktuell eine seperate PPSK SSID auf 2,4 GHz Basis für die ESP8266/ESP32 Gerätschaften, die ich so betreibe 😄 Das war der Trick.

**Comment by Freesee-10** [comment]
Wenn du ein Problem mit einem Update hast, kann ich aus bester Erfahrung folgendes Vorgehen empfehlen:
Trenne den DTU kurzfristig für mindestens 10 Sekunden vom Strom. Sobald es wieder mit dem Netz verbunden ist, versuchst du das Flaschen. Bei mir hat es nach diesem HW-Reset immer geklappt, dass das Flashen geklappt hat.

**Reply by dumpfheimer** [reply]
Danke! Das hat funktioniert! Einfacher Neustart scheint nichts zu ändern aber nach dem Trennen vom Netz hat das Update funktioniert.
